src/sentry/tasks/derive_code_mappings.py

```diff
@@ -192,4 +192,4 @@ def report_project_codemappings(
         msg = "derive_code_mappings: code mappings already exist."
         extra["existing_code_mappings"] = existing_code_mappings
 
-    logger.info(msg, extra)
+    logger.info(msg, extra=extra)
```

Comment:
This is the actual fix. GCP logs were not including the extra value.

In normal output you would see something like: `logger.info: Some message (extra={Json values})`